src/dtos/user.get.response.dto.ts

```diff
@@ -0,0 +1,36 @@
+import { IsNotEmpty, IsNumber, IsString, MaxLength } from 'class-validator';
+import AppointmentDto from './appointment.dto';
+import { Role } from '../entities';
+
+export default class UserResponseDto {
+  @IsNumber()
+  @MaxLength(50)
```

Comment:
@maxLength(50) is not compatible with @isNumber() because it is used for strings, not numbers; also, we should remove the password from a response DTO and I recommend using a special decorator for this type of DTO like @Exclude() and @Expose()
```suggestion
  @Exclude()
export default class UserResponseDto {
  @Expose()
  id: number;

  @Expose()
  name: string;

  @Expose()
  phone: string;

  @Expose()
  email: string;

  @Expose()
  role: Role;

  @Expose()
  @Type(() => AppointmentDto)
  appointments: AppointmentDto[];
}
```